packages/frontend/src/utils/font.tsx

```diff
@@ -0,0 +1,9 @@
+export const font = {
+    mainFont: "Open Sans', sans-serif;",
+    HeaderFontMobile: "700 32px/32px 'Open Sans'",
```

Comment:
czemu stosujesz jednostke 32px/32px? dlaczego za każdym razem podajesz że to Open Sans? Niektóre właściwości tekstu się np. font-family

Reply:
dobrze by było zrobić plik FontFamily gdzie możesz trzymać wszystkie rodzaje font famili.

Reply:
rozbiłam na pojedyncze pliki FontSize, FontWeigt FontFamily